libp2p/Capability.h

```diff
@@ -39,13 +28,12 @@ class CapabilityFace
     virtual unsigned version() const = 0;
     /// Number of messages supported by the subprotocol version.
     virtual unsigned messageCount() const = 0;
+    /// Time interval to run the background work loop at
+    virtual std::chrono::milliseconds backgroundWorkInterval() const = 0;
```

Comment:
What's the benefit of this? I intentionally went with this approach because it made more conceptual sense to me to have the host "pull" this value because it's managing capability execution. I'd be happy to go with another approach if it makes more sense though 😄